web_log_followers_change/README.rst

```diff
@@ -0,0 +1,4 @@
+Description
```

Comment:
Hi @cmsalmeida 
Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
Please create a proper README file following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst